MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1368,37 +1367,6 @@
 			shellPath = /bin/sh;
 			shellScript = "/usr/local/bin/carthage copy-frameworks";
 		};
-		35C7E0931E67AD80003ACB96 /* ShellScript */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-				"$(SRCROOT)/Carthage/Build/iOS/Mapbox.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/MapboxDirections.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/SDWebImage.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/Polyline.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/AWSPolly.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/AWSCore.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/Turf.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/Solar.framework",
```

Comment:
Does the developer need to copy these dependencies in their own application now? If so, we should update the installation instructions – this is a significant change, given the number of dependencies.

Reply:
Yes, but this has always been a requirement when using Carthage, and it's stated in their documentation when building for iOS, tvOS, or watchOS.

IIRC, I added this phase because we were unable to test frameworks without an application target but this doesn't seem to be an issue anymore.

Reply:
Yeah, I’m just thinking we should list these dependencies in the readme so developers know which ones are for the navigation SDK. It isn’t obvious that this SDK depends on AWSCore, for example.